lib/activerecord-multi-tenant/multi_tenant.rb

```diff
@@ -7,15 +7,6 @@ class Current < ::ActiveSupport::CurrentAttributes
     attribute :tenant
   end
 
-  def self.tenant_klass_defined?(tenant_name, options = {})
-    class_name = if options[:class_name].present?
-                   options[:class_name]
-                 else
-                   tenant_name.to_s.classify
-                 end
-    !!class_name.safe_constantize
```

Comment:
The class was dynamically loaded in this line.
Although top-level model can be checked with `const_defined?(:X)` or `autoload?(:X)`, it is difficult for nested models such as `A::B::Tenant`, so I gave up the method of checking by the presence or absence of a class.